templates/index.json

```diff
@@ -84,7 +84,7 @@
         "products_to_show": 8,
         "columns_desktop": 4,
         "color_scheme": "background-1",
-        "show_view_all": false,
+        "show_view_all": true,
         "swipe_on_mobile": false,
         "image_ratio": "adapt",
         "show_secondary_image": true,
```

Comment:
I see the default is `false` for `show_view_all` in the collection-list.liquid

Should we change it there as well?

Reply:
Good question. I would limit the scope of this PR to featured collection only, but we should consider if behaviour should be updated for collection list as well. cc: @Oliviammarcello